webpack.config.js

```diff
@@ -107,7 +107,32 @@ module.exports = (env, argv) => {
                 }),
                 // This plugin creates a CycloneDX Software Bill of Materials containing an aggregate of all bundled dependencies.
                 // It needs to be deactivated in watch mode
-                !argv.watch && new CycloneDxWebpackPlugin(cycloneDxWebpackPluginOptions)
+                !argv.watch && new CycloneDxWebpackPlugin(cycloneDxWebpackPluginOptions),
+                // This plugin will raise an error if a client file tries to import a server file
+                {
+                    apply(compiler) {
+                        const serverFiles = path.resolve(__dirname, 'src/server')
+                        compiler.options.resolve.plugins ??= []
+                        compiler.options.resolve.plugins.push({
+                            apply(resolver) {
+                                resolver.getHook('resolve').tapAsync('PreventServerInClient', async (request, _, callback) => {
+                                    if (!request.request || !request.context.issuer)
```

Comment:
We always surround statements (even single line) with curly braces. 
Maybe we should update the linter